src/components/accessibility/screen_reader_live/screen_reader_live.tsx

```diff
@@ -36,19 +37,30 @@ export interface EuiScreenReaderLiveProps {
    * `aria-live` attribute for both live regions
    */
   'aria-live'?: AriaAttributes['aria-live'];
+  /**
+   * Adds a tabindex={-1} to containing div. Focus is set on first
+   * component render and updates to `children` prop.
+   */
+  isFocusable?: boolean;
 }
 
 export const EuiScreenReaderLive: FunctionComponent<EuiScreenReaderLiveProps> = ({
   children,
   isActive = true,
   role = 'status',
   'aria-live': ariaLive = 'polite',
+  isFocusable = false,
 }) => {
   const [toggle, setToggle] = useState(false);
+  const focusRef = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
     setToggle((toggle) => !toggle);
-  }, [children]);
+
+    if (focusRef.current !== null && isFocusable) {
+      focusRef.current.focus();
+    }
+  }, [children, isFocusable]);
```

Comment:
Unless I'm missing something, this should probably be a separate effect in order to avoid unnecessarily toggling on focus:

```suggestion
  useEffect(() => {
    setToggle((toggle) => !toggle);
  }, [children]);

  useEffect(() => {
    if (focusRef.current !== null && isFocusable) {
      focusRef.current.focus();
    }
  }, [isFocusable]);

Reply:
I think you're right, breaking this up to separate `useEffect()` calls will prevent extra toggles. I'll catch it first thing in the morning.